src/timestamp.js

```diff
@@ -0,0 +1,5 @@
+
+export default function secondsToTimestamp(seconds) {    //formats seconds into mm:ss if less than an hour, hh:mm:ss if greater than an hour
```

Comment:
Looks good! The only thing I would change is to declare timeString as a const, as it is never being modified.

Reply:
Thanks! I'll make sure to fix that